editor/editor_spin_slider.cpp

```diff
@@ -365,6 +373,14 @@ String EditorSpinSlider::get_label() const {
 	return label;
 }
 
+void EditorSpinSlider::set_suffix(const String &p_suffix) {
+	suffix = p_suffix;
+	update();
+}
+String EditorSpinSlider::get_suffix() const {
```

Comment:
```suggestion
}

String EditorSpinSlider::get_suffix() const {
```

Reply:
This is a hassle to track manually, I wonder if an issue exists against Clang Format to be able to do this.

Reply:
If not, I can likely code this with a RegEx.